### PR TITLE
add support for setting a system scope

### PR DESCRIPTION
**Pull request by askarsabyrov**
Fixes: https://github.com/gophercloud/utils/issues/166
Reopened PR https://github.com/gophercloud/utils/pull/167

### CODE REVIEW

**Comment by askarsabyrov** [comment]
@pierreprinetti could you please approve workflow? really need this fix, it would help to use system scoped authorization for terraform-openstack-provider.

**Comment by mandre** [comment]
> @mandre hi, can you have a look please? really need this to be merged, thanks!

Sure, I'll check it soon. Sorry for the delay.

**Review by mandre — COMMENTED** [review]
Since you need to update the patch anyway, may I suggest you also credit @ignatenkobrain in the commit message as he was the one who wrote the original patch?
You can do it by adding a `Co-authored-by: Igor Raits <igor.raits@gooddata.com>` line at the end of the commit message (see the related [github docs](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)).

File: openstack/clientconfig/results.go [hunk 1]
```diff
@@ -78,6 +78,9 @@ type AuthInfo struct {
 	// Application Credential secret to login with.
 	ApplicationCredentialSecret string `yaml:"application_credential_secret,omitempty" json:"application_credential_secret,omitempty"`
 
+	// SystemScope is a system information to scope to.
+ 	SystemScope string `yaml:"system_scope,omitempty" json:"system_scope,omitempty"`
```
[the remark this answers is not in the document]
Comment (askarsabyrov) [reply openstack/clientconfig/results.go]: Removed space and added credits to @ignatenkobrain in the commit.

**Review by mandre — APPROVED** [review]
Looks good, thanks for the patch.

**Pull request merged**